.data/in/track-simplified.csv

```diff
@@ -0,0 +1,149 @@
+2018-10-25T17:10:25;52.301598;13.211489;102;14.1666666666667
+2018-10-25T17:10:30;52.301271;13.212073;160;11.6666666666667
+2018-10-25T17:10:33;52.300911;13.212025;196;11.9444444444444
+2018-10-25T17:10:35;52.300718;13.211752;234;13.0555555555556
+2018-10-25T17:10:39;52.30063;13.210904;270;15.5555555555556
+2018-10-25T17:10:50;52.300766;13.208967;274;5
+2018-10-25T17:10:56;52.300632;13.208683;192;5.83333333333333
+2018-10-25T17:11:45;52.293085;13.209536;175;15.2777777777778
+2018-10-25T17:11:58;52.291424;13.209112;185;13.6111111111111
+2018-10-25T17:12:12;52.289662;13.209118;178;13.8888888888889
+2018-10-25T17:12:16;52.289198;13.209214;164;11.6666666666667
+2018-10-25T17:12:29;52.287779;13.210132;154;12.2222222222222
+2018-10-25T17:12:54;52.285309;13.212057;170;11.1111111111111
+2018-10-25T17:13:32;52.282114;13.21099;178;9.44444444444444
+2018-10-25T17:14:00;52.278104;13.213012;178;9.44444444444444
+2018-10-25T17:14:09;52.277018;13.212315;192;18.6111111111111
+2018-10-25T17:14:46;52.269983;13.209391;194;20.2777777777778
+2018-10-25T17:14:59;52.267673;13.208635;199;19.4444444444444
+2018-10-25T17:15:31;52.262658;13.203995;202;14.4444444444444
+2018-10-25T17:15:39;52.261563;13.204129;181;16.9444444444444
+2018-10-25T17:15:50;52.259707;13.203802;180;15
+2018-10-25T17:16:12;52.2564;13.206355;163;15
+2018-10-25T17:16:17;52.255716;13.206468;178;15.2777777777778
+2018-10-25T17:16:29;52.25412;13.206119;175;15.2777777777778
+2018-10-25T17:16:34;52.253337;13.206511;158;17.2222222222222
+2018-10-25T17:16:57;52.251095;13.20901;137;13.6111111111111
+2018-10-25T17:17:16;52.249544;13.21158;143;8.61111111111111
+2018-10-25T17:17:23;52.248989;13.21099;194;11.6666666666667
+2018-10-25T17:17:30;52.247975;13.21129;167;18.8888888888889
+2018-10-25T17:17:35;52.247114;13.211794;154;21.3888888888889
+2018-10-25T17:17:42;52.245795;13.213087;143;23.0555555555556
+2018-10-25T17:18:02;52.242225;13.218897;130;27.5
+2018-10-25T17:18:21;52.239821;13.224792;122;18.3333333333333
+2018-10-25T17:18:32;52.239027;13.227389;130;15.5555555555556
+2018-10-25T17:18:47;52.238011;13.228489;158;15.5555555555556
+2018-10-25T17:18:52;52.23722;13.22866;180;18.3333333333333
+2018-10-25T17:19:06;52.234873;13.228408;188;16.9444444444444
+2018-10-25T17:19:17;52.233438;13.227754;184;16.6666666666667
+2018-10-25T17:19:27;52.232035;13.227882;182;11.1111111111111
+2018-10-25T17:19:36;52.231442;13.227614;184;8.33333333333333
+2018-10-25T17:19:40;52.231107;13.227813;170;11.3888888888889
+2018-10-25T17:20:47;52.222395;13.219101;184;7.5
+2018-10-25T17:21:03;52.220759;13.218103;195;11.6666666666667
+2018-10-25T17:21:20;52.218828;13.218012;185;11.9444444444444
+2018-10-25T17:21:55;52.218154;13.217513;185;6.38888888888889
+2018-10-25T17:22:01;52.217798;13.218033;122;10.8333333333333
+2018-10-25T17:22:16;52.216918;13.220125;122;10.5555555555556
+2018-10-25T17:22:54;52.215695;13.226246;113;7.22222222222222
+2018-10-25T17:24:54;52.215443;13.226262;53;0
+;;;
```

Comment:
I guess you didn't want to commit this file

Reply:
no, i wanted in my original idea but the cleaning was rather rapid.

Reply:
removed